### ISSUE TITLE
Images embedded from the camera are rotated 90 degrees on iOS

### ISSUE THREAD

**Issue by raefa** [AppWerft/TiJSPDF#9]
If you add an image that you have saved from the camera and embed it in a PDF, it rotates 90 degrees on iOS (not Android).

```
var imageFile = Ti.Filesystem.getFile(Ti.Filesystem.tempDirectory, "file-from-camera.png"); 
var tempFile = Ti.Filesystem.createTempFile(Ti.Filesystem.tempDirectory);
tempFile.write(ImageFactory.compress(imageFile.read(), 1));
pdf.addImage(tempFile.nativePath, 'JPEG', imageX, imageY, imageWidth, imageHeight);
```

As a workaround, I am adding the image to a view and using toImage() and creating a new image. It appears the right way up then!

**Comment by AppWerft** [comment]
It is a "problem" with iOS camera. The camera saves in right orientation and in EXIF part you can read the orientation. Maybe [this module](https://github.com/gstreetmedia/Titanium-Mobile-ExifData) ghelps you to extract the data.

**Comment by raefa** [comment]
Ahhh. Understood. I used that class to extract the orientation. I then rotated this image using ImageFactory (https://github.com/collinprice/ti.imagefactory/blob/stable/ios/documentation/index.md) ... but by rotating the image it strips the Exif/header data from the JPEG and you therefore can't embed it in the PDF. So I am back to my workaround from above - adding the image to a view and using toImage() and creating a new image. I am sharing this in case anyone else has this issue.
